server/controllers/user.js

```diff
@@ -46,6 +46,54 @@ class UserSign {
       },
     });
   }
+
+
+  static async signin(req, res) {
+    const emailget = 'SELECT * FROM users WHERE email =$1';
+    const { rows: [emailGot] } = await pool.query(emailget, [req.body.email]);
+    if (!emailGot) {
+      return res.status(422).json({
+        status: 422,
+        message: 'Invalid email address',
+      });
+    }
+
+    const getPassword = 'SELECT * FROM users  WHERE email = $1;';
+    const { rows: [passwordGot] } = await pool.query(getPassword, [req.body.email]);
+    
+    const password = await bcrypt.compare(req.body.password, passwordGot.password);
+    if (!password) {
+      return res.status(422).json({
```

Comment:
I suggest you read on HTTP response status codes and update your response accordingly. 
https://developer.mozilla.org/en-US/docs/Web/HTTP/Status

Reply:
thanks